### ISSUE TITLE
using with webpack 4.46.0

### ISSUE THREAD

**Issue by AlonsoK28** [digitalfortress-tech/typeahead-standalone#39]
### What happened?

I'm triying to use  `typeahead-standalone` in an existing `webpack 4.46.0` project and get the following error when `import` the lib

``` my-output-error.js
 error  in ./node_modules/typeahead-standalone/dist/typeahead-standalone.js

Module parse failed: Unexpected token (134:38)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|               C = e.source.identity || w,
|               x = e.onSubmit || n,
>               O = e.source.dataTokens?.constructor === Array ? e.source.dataTokens : void 0,
|               L = {},
|               N = {},
```

### Affected version

Latest

### Reproduction URL

none

### How to reproduce?

1. Use `webpack 4.46`
2. Import `typeahead-standalone` in your proyect
3. See the error on the console output

### Browsers

Chrome

### Relevant log output

```Shell
ERROR  Failed to compile with 1 errors                                                                                       23:23:58
 error  in ./node_modules/typeahead-standalone/dist/typeahead-standalone.js

Module parse failed: Unexpected token (134:38)
File was processed with these loaders:
 * ./node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|               C = e.source.identity || w,
|               x = e.onSubmit || n,
>               O = e.source.dataTokens?.constructor === Array ? e.source.dataTokens : void 0,
|               L = {},
|               N = {},

   844 modules
```

### Additional context

I'm using the following `ES6 modules` import statement according the documentation and get error

``` my-import-statement.js
// using ES6 modules
import typeahead from 'typeahead-standalone';
```

Triying to import directly from `dist` path also fails

``` my-other-import-statement.js
import typeahead from 'typeahead-standalone/dist/typeahead-standalone';
```

**Comment by niketpathak** [comment]
Hello @AlonsoK28 , Thank you for reporting this. I will look into it!

**Comment by AlonsoK28** [comment]
this is the [webpack 4.4.6](https://www.npmjs.com/package/webpack/v/4.46.0) package 😁

greetings

**Comment by craiggeil** [comment]
Yep just spent 4 hours trying to figure out why rollup wouldn't work with this library. This was it.

**Comment by AlonsoK28** [comment]
there any solution for this ☺

**Comment by niketpathak** [comment]
Hello @AlonsoK28 and @craiggeil , this issue has been fixed in v[4.14.1](https://www.npmjs.com/package/typeahead-standalone)
